packages/cms-cli/commands/secrets.js

```diff
@@ -24,7 +28,8 @@ function configureSecretsCommand(program) {
     .version(version)
     .description('Manage secrets')
     .command('add <secret-name> <secret-value>', 'add a HubSpot secret')
-    .command('delete <secret-name>', 'delete a HubSpot secret');
+    .command('delete <secret-name>', 'delete a HubSpot secret')
```

Comment:
Since the positional args are within the context of the secrets command, I think we can shorted their slug names to just `name` and `value`.